### PR TITLE
Rocco/add car

### PR DESCRIPTION
**Pull request by yusufroqib**

### CODE REVIEW

File: frontend/src/app/dashboard/layout.tsx
Comment (michojekunle) [inline frontend/src/app/dashboard/layout.tsx:1]: Amazing Job asbtracting these page into components Rocco 🚀

**Review by michojekunle — APPROVED** [review]
Awesome stuff, Rocco, You did an amazing Job

File: frontend/src/app/dashboard/layout.tsx
Comment (michojekunle) [reply frontend/src/app/dashboard/layout.tsx:1]: Did you run build before pushing, There's a build error can you look into that and fix before the merge
image

Comment (yusufroqib) [reply frontend/src/app/dashboard/layout.tsx:1]: Thanks. I have fixed that. 
Screenshot 2024-10-11 181503

Comment (michojekunle) [reply frontend/src/app/dashboard/layout.tsx:1]: Awesome awesome 🔥🚀

**Pull request merged**